manta-crypto/src/merkle_tree/partial.rs

```diff
@@ -20,16 +20,249 @@
 
 use crate::merkle_tree::{
     capacity,
-    inner_tree::{BTreeMap, InnerMap, PartialInnerTree},
+    inner_tree::{BTreeMap, InnerMap, InnerNode, PartialInnerTree},
     Configuration, CurrentPath, InnerDigest, Leaf, LeafDigest, MerkleTree, Node, Parameters, Path,
     PathError, Root, Tree, WithProofs,
 };
 use alloc::vec::Vec;
 use core::{fmt::Debug, hash::Hash};
 
+#[cfg(feature = "std")]
+use std::collections::hash_map::HashMap;
+
 #[cfg(feature = "serde")]
 use manta_util::serde::{Deserialize, Serialize};
```

Comment:
`serde` comes before `std` alphabetically:

```suggestion
#[cfg(feature = "serde")]
use manta_util::serde::{Deserialize, Serialize};

#[cfg(feature = "std")]
use std::collections::hash_map::HashMap;
```